tests/js/spec/views/dashboardsV2/detail.spec.jsx

```diff
@@ -461,6 +463,61 @@ describe('Dashboards > Detail', function () {
       expect(wrapper.find('AddWidget').exists()).toBe(true);
     });
 
+    it('opens custom modal when add wiget option is clicked', async function () {
```

Comment:
Small thing but some of these tests have `wiget` in the title, can we swap them for `widget`?